client/Cargo.toml

```diff
@@ -11,7 +11,9 @@ keywords = ["trillium", "framework", "async"]
 categories = ["web-programming", "web-programming::http-client"]
 
 [features]
-json = ["serde_json", "serde", "thiserror"]
+default = ["cookies", "json"]
```

Comment:
Please consider making `cookies` not be enabled by default. Based on previous experience with `tide`, once crates in the ecosystem start depending on `trillium-client`, it's really hard to get all of them using `default-features = false`, even though many won't need this.

I know that trillium is going for a "just works" API, but having to enable a feature flag to get a more featureful client doesn't seem like *too* much to ask here.

Reply:
That's a really useful bit of feedback! My initial inclination was for each client handler be their own crate to make totally clear that it doesn't depend on any private state, but convinced myself to limit the proliferation of tiny `trillium-client-*` crates. I think I may have overcorrected in the other direction. I'll change them to opt-in instead of opt-out before release, or possibly even go for the trillium-client-cookies, trillium-client-follow-redirects, trillium-client-cache crates

Reply:
@jbr Perhaps a middle ground might make sense: a single `trillium-client-handlers` crate for all the handlers that *don't* add dependencies, and either a separate crate or feature flag (or both with re-export) for handlers that *do* add dependencies.